docs/manage-stateHandle-in-sessionStorage.org

```diff
@@ -2,6 +2,15 @@
 @startuml
 start
 
+note left
```

Comment:
We should ask devex team to update this related to their `interactionCode` flow as well

Reply:
@aaronbrodersen-okta @shuowu ^^

Reply:
@aarongranick-okta correct Aaron.